radar/data_management/elasticsearch.yaml

```diff
@@ -1,9 +1,13 @@
 name: Elasticsearch
 blip:
   - date: 2018-03-20
-    ring: TRIAL
+    ring: ADOPT
```

Comment:
Add a new date entry with ring to preserve history.

```
- date: 2018-03-20
  ring: TRIAL
- date: 2019-05-23
  ring: ADOPT
```

Reply:
I have updated the PR with the requested changes.